shared/js/background/devbuild-reloader.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Reloads the extension when changes are detected. Only included for developer
+ * builds. (Pass the reloader=0 build parameter to disable.)
+ */
+
+import browser from 'webextension-polyfill'
+const browserWrapper = require('./wrapper')
+
+function createAlarm () {
+    browserWrapper.createAlarm('checkBuildTime', { when: Date.now() + 5000 })
+}
+
+browser.alarms.onAlarm.addListener(async alarmEvent => {
+    if (alarmEvent.name !== 'checkBuildTime') {
+        return
+    }
```

Comment:
Could this conflict with the centralized alarm listener in events.js? Does the alarm trigger for all listeners?

Reply:
No, since the alarm name is checked in all the listeners.